collectors/googlestackdriver/README.md

```diff
@@ -0,0 +1,101 @@
+# Googlestackdriver collector
+Alert Logic Googlestackdriver AWS Based API Poll (PAWS) Log Collector Library.
+
+# Overview
+This repository contains the AWS JavaScript Lambda function and CloudFormation 
+Template (CFT) for deploying a log collector in AWS which will poll 3rd party service API to collect and 
+forward logs to the Alert Logic CloudInsight backend services.
+
+# Installation
+
+Refer to [CF template readme](./cfn/README.md) for installation instructions.
+
+## Generate Google API key file
+1. In the Google Cloud Console, go to the Create service account key page.
+2. From the Service account list, select New service account.
+3. In the Service account name field, enter a name.
+4. From the Role list, select Project > Owner.
+
+Note: The Role field authorizes your service account to access resources. You can view and change this field later by using the Cloud Console. If you are developing a production app, specify more granular permissions than Project > Owner. For more information, see granting roles to service accounts.
+
+5. Click Create. A JSON file that contains your key downloads to your computer.
```

Comment:
Nothing actually wrong here, but wondering if this could be automated? Seeing the word "Service" in reference to a collector reminded me of Azure and filled me with dread

Reply:
It is similar to Azure, but mich more straightforward. Users would still need to log into their consoles, but once its created, its simply a matter of downloading the JSON and using it in the collector.